### PR TITLE
Nuclearity

### PR DESCRIPTION
**Pull request by brookwander**

### CODE REVIEW

File: catlas/nuclearity.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    return hist
+
+
+def evaluate_infiniteness(hist, hist_rep):
+    """
+    Function that compares the connected counts between the minimal slab
+    and a repeated slab to classify the type of infiniteness.
+    Args:
+        hist: counts of neighbor types in the minimal slab
+        hist: counts of neighbor types in the minimal slab
```
Comment (KirbyBroderick) [inline catlas/nuclearity.py:89]: typo

File: catlas/nuclearity.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    labels, hist = topology.label_components(graph, directed=False)
+    return hist
+
+
+def evaluate_infiniteness(hist, hist_rep):
+    """
+    Function that compares the connected counts between the minimal slab
+    and a repeated slab to classify the type of infiniteness.
+    Args:
+        hist: counts of neighbor types in the minimal slab
```
Comment (KirbyBroderick) [inline catlas/nuclearity.py:88]: Is this a list of lists of ints?

File: catlas/nuclearity.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import graph_tool as gt
+from graph_tool import topology
+import numpy as np
+
+
+def get_nuclearity(entry):
+    """
+    Function to get the nuclearity for each element in a surface
+    Args:
+        entry: a catlas-like entry object
```
[the remark this answers is not in the document]
Comment (brookwander) [reply catlas/nuclearity.py:18]: a dictionary with catlas-like entries. Can change if you have a preference!

File: catlas/nuclearity.py (hunk 2 above)
Comment (brookwander) [reply catlas/nuclearity.py:88]: its a list of ints. I will make that more obvious. Thats why I take the max() of it

**Pull request merged**